.github/workflows/storybook-tests.yml

```diff
@@ -22,9 +22,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install
       - name: Install Playwright
-        run: pnpm install playwright -g
+        run: pnpm install playwright@1.51.1 -g
       - name: setup playwright
-        run: npx playwright install --with-deps
+        run: npx playwright@1.51.1 install --with-deps
```

Comment:
I realized we don't need this version here. Only the change above. Sorry for creating the conflict